vite.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'vite';
```

Comment:
Unfortunately if using bun the cache likes to be tested, can remove it with config.

```suggestion
import { defineConfig } from 'vitest/config';
```

```suggestion
{
...
test: {
  exclude: ['*/~/**']
}
}

Reply:
Ok, great idea

Reply:
https://github.com/oven-sh/bun/issues/4145

When running vitest with bun it just crashes in a fire.
I excluded the test from the config. so it does not catch fire